test/bin/install-mssql.ps1

```diff
@@ -0,0 +1,31 @@
+$ProgressPreference = 'SilentlyContinue'
+
+if (-not(Test-path "C:\Downloads"))
+{
+    mkdir "C:\Downloads"
+}
+
+$sqlInstallationFile = "C:\Downloads\sqlexpress.exe"
+if (-not(Test-path $sqlInstallationFile -PathType leaf))
+{
+    Write-Host "Downloading SQL Express ..."
+    Invoke-WebRequest -Uri "https://go.microsoft.com/fwlink/?linkid=829176" -OutFile "C:\Downloads\sqlexpress.exe"
+}
+
+Write-Host "Installing SQL Express ..."
```

Comment:
I definitely agree it'd be great to test _on Windows_ (especially since the job name is already "test windows"... oops... 🤦‍♂️). However, installing SQL Server is SUCH a headache. Are we able to use Docker for this? I don't think the Windows container is available any longer, so we'd need to use the Linux image, which ought to be possible by connecting to it via setup_remote_docker, I think

Reply:
Yeah the Windows container are no longer available. But also Windows containers were quite a pain to set up, I can see why it was easier for Microsoft to just push WSL2 😁 

`https://circleci.com/docs/building-docker-images/#accessing-services` doesn't allow [to access services from the primary executor](https://circleci.com/docs/building-docker-images/#accessing-services). It looks like it's a detached machine somewhere else in the CircleCI cloud.

Reply:
That said, I use a similar version of this script at work and has been working without any changes for more than a year now. I also use it for SQL server 2019 and 2022, so in case the pipeline should be extended at some point to test across different SQL server versions, it should be possible with little modification.